ui/components/FilterDialog.tsx

```diff
@@ -37,16 +37,12 @@ const SlideContent = styled.div`
 export const filterSeparator = ":";
 
 const FilterSection = ({ header, options, formState, onSectionSelect }) => {
-  const [all, setAll] = React.useState(false);
-  React.useEffect(() => {
-    const allChecked = _.chain(formState)
-      // get all relevant keys' current value
-      .keys()
-      .filter((key) => _.includes(key, header))
-      .every((key) => formState[key])
-      .value();
-    setAll(allChecked);
-  });
+  const all = _.chain(formState)
+    // get all relevant keys' current value
+    .keys()
+    .filter((key) => _.includes(key, header))
+    .every((key) => formState[key])
```

Comment:
`every` looks sus? 🤔

Reply:
```
_.every([], () => true)
true
```

Reply:
Yeah... I've found it a bit difficult to get my head around it so rewrote it